DotCMIS/DotCMIS.csproj

```diff
@@ -40,6 +40,10 @@
     <AssemblyOriginatorKeyFile>dotcmis.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="log4net, Version=1.2.13.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\log4net.2.0.3\lib\net40-full\log4net.dll</HintPath>

```

Comment:
This path does not exist on computers other than yours. Embedding the log4net DLL (for instance at the root of the project) could be a solution.